go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/hashicorp/go-version v1.6.0
 	github.com/jpillora/backoff v1.0.0
 	github.com/json-iterator/go v1.1.12
-	github.com/kopia/kopia v0.13.1-0.20230605143225-3551f743d762
```

Comment:
Ignore the changes in go.mod and go.sum, these are from Le's PR - https://github.com/kanisterio/kanister/pull/2416